app/views/layouts/_header.html.erb

```diff
@@ -1,78 +1,79 @@
+    <%= link_to(content_tag(:span, ArchiveConfig.APP_NAME) + content_tag(:sup, " beta") +
+    image_tag("/images/ao3_logos/logo_42.png", alt: ArchiveConfig.ALT_LOGO, class: "logo", skip_pipeline: true), root_path) %>
   </h1>
 
   <% if logged_in? || logged_in_as_admin? %>
-    <%= render 'users/sessions/greeting' %>
+    <%= render "users/sessions/greeting" %>
   <% else %>
     <div id="login" class="dropdown">
-      <p class="user actions" role="menu">
-        <%= link_to t(".login"), new_user_session_path, id: "login-dropdown", role: "menuitem" %>
+      <p class="user actions">
+        <%= link_to t(".login"), new_user_session_path, id: "login-dropdown" %>
       </p>
-      <%= render 'users/sessions/login' %>
+      <%= render "users/sessions/login" %>
     </div>
   <% end %>
 
-  <h3 class="landmark heading"><%= ts('Site Navigation', key: 'header') %></h3>
-  <ul class="primary navigation actions" role="navigation">
-    <li class="dropdown">
-      <%= link_to ts('Fandoms', key: 'header'), menu_fandoms_path %>
-      <%= render 'menu/menu_fandoms' %>
-    </li>
-    <li class="dropdown">
-      <%= link_to ts('Browse', key: 'header'), menu_browse_path %>
-      <%= render 'menu/menu_browse' %>
-    </li>
-    <li class="dropdown">
-      <%= link_to ts('Search', key: 'header'), menu_search_path %>
-      <%= render 'menu/menu_search' %>
-    </li>
-    <li class="dropdown">
-      <%= link_to ts('About', key: 'header'), menu_about_path %>
-      <%= render 'menu/menu_about' %>
-    </li>
-	  <li class="search"><%= render 'works/search_box' %></li>
-  </ul>
+  <nav aria-label="<%= t(".nav.label") %>">
+    <ul class="primary navigation actions">
+      <li class="dropdown">
+        <%= link_to t(".nav.fandoms"), menu_fandoms_path %>
+        <%= render "menu/menu_fandoms" %>
+      </li>
+      <li class="dropdown">
+        <%= link_to t(".nav.browse"), menu_browse_path %>
+        <%= render "menu/menu_browse" %>
+      </li>
+      <li class="dropdown">
+        <%= link_to t(".nav.search"), menu_search_path %>
+        <%= render "menu/menu_search" %>
+      </li>
+      <li class="dropdown">
+        <%= link_to t(".nav.about"), menu_about_path %>
+        <%= render "menu/menu_about" %>
+      </li>
+      <li class="search"><%= render "works/search_box" %></li>
+    </ul>
+  </nav>
 
   <% if logged_in_as_admin? %>
-    <%= render 'admin/header' %>
+    <%= render "admin/header" %>
   <% end %>
 
   <% if @collection %>
     <h2 class="collections">
       <% unless @collection.header_image_url.present? %>
-        <%= @collection.title ? @collection.title : (@collection_parent ? @collection_parent.title : ts('New Collection', key: 'header')) %>
+        <%= if @collection.title
+              @collection.title
+            elsif @collection_parent
+              @collection_parent.title
+            else
+              t(".collections.new")
+            end %>
```

Comment:
I much prefer this over the nested ternary, but if you want to keep it a 1-liner:
```suggestion
        <%= @collection.title || @collection_parent&.title ||  t(".collections.new") %>
```